examples/config-basic.json

```diff
@@ -1,7 +1,4 @@
 {
```

Comment:
Shouldn't the `dotenv` config be in here as well? It's a pretty fundamental part of the 'basic' thing as well.